src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -109,9 +109,9 @@
     protected static BeaconManager client = null;
     private final ConcurrentMap<BeaconConsumer, ConsumerInfo> consumers = new ConcurrentHashMap<BeaconConsumer,ConsumerInfo>();
     private Messenger serviceMessenger = null;
-    protected RangeNotifier rangeNotifier = null;
+    protected final List<RangeNotifier> rangeNotifiers = new CopyOnWriteArrayList<>();
```

Comment:
We do need to keep `getMonitorNotifier()` and `getRangeNotifier()` so that this is not a breaking API change.  I think a `CopyOnWriteArraySet` is probably fine.